announcements/1.13.0.md

```diff
@@ -0,0 +1,93 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.13.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### OpenGL ES 3.2 support
+
+After supporting OpenGL ES 3.1 only for a long time Anbox Cloud finally adds full OpenGL ES 3.2 support
+for NVIDIA GPUs and software rendering. This provides a increased compatibility with available Android
+games.
+
+### Direct rendering for AMD and Intel GPUs
+
+Since the beginning of Anbox Cloud graphics APIs were exposed to the Android container through an
+translation layer and actual execution of commands happened on the outside. With 1.13, Anbox Cloud
+now enables direct GPU access inside the Android container for AMD and Intel GPUs based on the
+open source [Mesa driver](https://mesa3d.org/). This provides better performance and stability and
+increased compatibility.
```

Comment:
```suggestion
Since the beginning of Anbox Cloud, graphics APIs were exposed to the Android container through a
translation layer, and the actual execution of commands happened on the outside. With 1.13, Anbox Cloud
enables direct GPU access inside the Android container for AMD and Intel GPUs based on the
open source [Mesa driver](https://mesa3d.org/). This direct access provides better performance and stability and
increases compatibility.
```